src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -20,6 +20,8 @@
 ## Upcoming Release
 * Azure Site Recovery bug fixes for VMware to Azure Reprotect, Update policy and Disable scenarios.
 * Azure Backup added the support for UserAssigned MSI in RecoveryServices Vault.
+* Added support for ZRS disk type for Azure to Azure replication.
+* Added Availability zone information in replicated protected item response for Azure to Azure replication.
```

Comment:
Please add the changelog after line 20. Or it will conflict when our release branch is merged back.

Reply:
Updated